### PR TITLE
New MySQL view behaviour for stretch guests

### PR DESCRIPTION
**Pull request by abrahammartin**
It should't allow to change root mysql password as in stretch because users are added as mysql admins and have all permisions granted.
- New screen for stretch based VMs redirects to mws documentation.
- Change db root pwd view redirects to the main panel if user tries to access anyway.
- No changes for jessie based VMs.

### CODE REVIEW

File: mws/sitesmanagement/views/others.py [hunk 1]
```diff
@@ -125,10 +125,13 @@ def service_settings(request, service_id):
                 url=reverse(service_settings, kwargs={'service_id': service.id}))
     }
 
+    service_conf = AnsibleConfiguration.objects.filter(service=service, key='os')
```
Comment (rjw57) [inline mws/sitesmanagement/views/others.py]: Why can we not use ``service.ansible_configuration.os`` directly in the template if we're doing an extra DB call anyway?

File: mws/templates/mws/settings.html [hunk 2]
```diff
@@ -149,7 +149,8 @@ <h3 class="campl-teaser-title">Unix Groups</h3>
 
             <div class="campl-column3">
                 <div class="campl-content-container campl-side-padding">
-                    <a href="{% url 'change_db_root_password' service_id=service.id %}">
+                    <a href="{% if os == 'stretch' %}https://mws-help.uis.cam.ac.uk/provided-software/mysql{%
```
Comment (rjw57) [inline mws/templates/mws/settings.html]: Could use a ``{% with os=service.ansible_configuration.os %}`` instead?

File: mws/sitesmanagement/views/others.py (hunk 1 above)
Comment (abrahammartin) [reply mws/sitesmanagement/views/others.py]: good point

**Comment by abrahammartin** [comment]
changes done

**Pull request merged**